package/MDAnalysis/analysis/pca.py

```diff
@@ -106,8 +106,13 @@ class PCA(AnalysisBase):
 
     After initializing and calling method with a universe or an atom group,
     principal components ordering the atom coordinate data by decreasing
-    variance will be available for analysis. Please refer to the
-    :ref:`PCA-tutorial` for more detailed instructions.
+    variance will be available for analysis. As an example:
+        >>> pca = PCA(atomgroup, select='backbone').run()
```

Comment:
I use in my code blocks

```
.. code::python
  >>> # now comes the code
```

Have a look around for this in the other doc strings and you see what I mean.